tasks/packages.yml

```diff
@@ -2,10 +2,8 @@
 ---
 - name: packages | copy r scripts
   template:
-    src: "usr/local/bin/{{ item.src }}"
-    dest: "/usr/local/bin/{{ item.dest }}"
-    owner: root
-    group: root
```

Comment:
Why remove `owner` and `group`?

Reply:
because you may want to run the ansible task as another user

Reply:
Sure, but do you want the script to be owned by someone else?

Reply:
yes it is, see my comment here: https://github.com/Oefenweb/ansible-r/pull/52#issuecomment-811980474